spec.html

```diff
@@ -8637,8 +8663,9 @@ <h1>ModuleNamespaceCreate ( _module_, _exports_ )</h1>
           1. Assert: _module_ is a Module Record.
           1. Assert: _module_.[[Namespace]] is *undefined*.
           1. Assert: _exports_ is a List of String values.
-          1. Let _M_ be a newly created module namespace exotic object with the internal slots listed in <emu-xref href="#table-29"></emu-xref>.
+          1. Let _M_ be ! AllocateBasicObject(&laquo; [[Module]], [[Exports]], [[Prototype]] &raquo;).
```

Comment:
one change this seems to cause is that Table 29 is no longer the sole source of truth for these slots. Can you think of a way to avoid listing them twice?

Reply:
Well, you could say something like:
```
4. Let _internalSlotsList_ be the names of the internal slots listed in [Table 29].
5. Let _M_ be ! AllocateBasicObject(_internalSlotsList_).
```
but I don't know about "sole source of truth". If someone added a slot to Table 29, ModuleNamespaceCreate would still have to name it to set it.

Reply:
Although it's not perfect, @jmdyck's version seems nice; I'll do that.